modules/core/04-channel/v2/keeper/grpc_query_test.go

```diff
@@ -586,3 +586,134 @@ func (suite *KeeperTestSuite) TestQueryNextSequenceSend() {
 		})
 	}
 }
+
+func (suite *KeeperTestSuite) TestQueryUnreceivedAcks() {
+	var (
+		req    *types.QueryUnreceivedAcksRequest
+		expSeq = []uint64{}
+	)
+
+	testCases := []struct {
+		msg      string
+		malleate func()
+		expError error
+	}{
+		{
+			"success",
+			func() {
+				path := ibctesting.NewPath(suite.chainA, suite.chainB)
+				path.SetupV2()
```

Comment:
[nit] should the v2 path be created in the main test body and not need to do it in the malleate fns?